### DISCUSSION TITLE
Installation problem on Windows

### DISCUSSION THREAD

**Discussion by canu-ugur** [alan-turing-institute/PeriPy#126] [Q&A]
Hello, I've been trying to install peripy code on my Windows computer. I've followed required steps and obtained all dependencies and requirements. However, the installation is interrupted when I tried to run the command: _pip install peripy_

I got the following error message which is between %% ... %%  symbols. Any help would be appreciated. 
Thanks
Ugur Can

%%
[peripy_ERROR.txt](https://github.com/alan-turing-institute/PeriPy/files/7962352/peripy_ERROR.txt)

    ERROR: Command errored out with exit status 1:
     command: 'C:\Anaconda3\python.exe' -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\canu\\AppData\\Local\\Temp\\pip-install-lz8u8393\\peripy_2a9a90bf3bd14378ae000b51e74d7ff7\\setup.py'"'"'; __file__='"'"'C:\\Users\\canu\\AppData\\Local\\Temp\\pip-install-lz8u8393\\peripy_2a9a90bf3bd14378ae000b51e74d7ff7\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\canu\AppData\Local\Temp\pip-pip-egg-info-rkkimb3q'
         cwd: C:\Users\canu\AppData\Local\Temp\pip-install-lz8u8393\peripy_2a9a90bf3bd14378ae000b51e74d7ff7\
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\canu\AppData\Local\Temp\pip-install-lz8u8393\peripy_2a9a90bf3bd14378ae000b51e74d7ff7\setup.py", line 71, in <module>
        ext_modules=cythonize(ext_modules),
      File "C:\Anaconda3\lib\site-packages\Cython\Build\Dependencies.py", line 965, in cythonize
        module_list, module_metadata = create_extension_list(
      File "C:\Anaconda3\lib\site-packages\Cython\Build\Dependencies.py", line 815, in create_extension_list
        for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
      File "C:\Anaconda3\lib\site-packages\Cython\Build\Dependencies.py", line 114, in nonempty
        raise ValueError(error_msg)
    ValueError: 'peripy/create_crack.pyx' doesn't match any files
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/bb/d8/0649d03f630104591b18da77bb233a6c949831108b880078bea802abff42/peripy-1.0.0.tar.gz#sha256=b88874ae777454a794700a2b105713d47869c302d9201c058528dd9f16c759cb (from https://pypi.org/simple/peripy/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement peripy (from versions: 1.0.0)
ERROR: No matching distribution found for peripy

%%

**Comment by alphaDeng** [comment]
maybe install from source will fix it?
